src/views/SpaceProposal.vue

```diff
@@ -214,6 +214,24 @@ onMounted(async () => {
     clickVote();
   }
 });
+
+const showFullMarkdownBody = ref(false);
+
+// Scroll to top of the page after clicking "Show less" button
+watch(showFullMarkdownBody, () => {
+  if (!showFullMarkdownBody.value) window.scrollTo(0, 0);
+});
+
+// Ref to the proposal body element
+const markdownBody = ref(null);
+
+// Detect if the proposal body is too long and should be shortened
+const truncateMarkdownBody = computed(() => {
+  const markdownBodyHeight = markdownBody.value?.clientHeight
+    ? markdownBody.value.clientHeight
+    : 0;
+  return markdownBodyHeight > 280 ? true : false;
```

Comment:
I would recommend making the visible body bigger, like 360px height
```suggestion
  return markdownBodyHeight > 360 ? true : false;
```

Reply:
Yes works for me